seq/src/test/java/com/jmonad/seq/SeqDropTest.java

```diff
@@ -0,0 +1,16 @@
+package com.jmonad.seq;
+
+import org.junit.Test;
+
+public class SeqDropTest {
+
+  @Test public void dropListElementsTest() {
+    Integer[] numbers = {1, 2, 3, 4, 5, 6, 7, 8, 9, 10};
+
+    Seq<Integer> elements = new Seq<Integer>(numbers);
+
+    assert elements.drop(5).toArrayList().toString().equals("[6, 7, 8, 9, 10]");
```

Comment:
I think you can split this assertions into other tests. Make just one assertion in each test, and use a self explanatory name.. saying what you're testing, the input and the expected.. Sometimes you're test expect an error, and you should explain it in the test. ok?